code/modules/library/skill_learning/skillchip.dm

```diff
@@ -97,33 +97,36 @@
  * Arguments:
  * * silent - Boolean. Whether or not an activation message should be shown to the user.
  * * force - Boolean. Whether or not to just force de-activation if it would be prevented for any reason.
+ * * brain_owner - the owner var of the brain is set to null on organ/on_mob_remove(), so we need this if owner is null.
  */
-/obj/item/skillchip/proc/try_deactivate_skillchip(silent = FALSE, force = FALSE)
+/obj/item/skillchip/proc/try_deactivate_skillchip(silent = FALSE, force = FALSE, mob/living/brain_owner)
 	if(!active)
 		return "Skillchip is not active."
 
 	// Should not happen. Holding brain is destroyed and the chip hasn't had its state set appropriately.
 	if(!holding_brain)
-		stack_trace("Skillchip's owner is null or qdeleted brain.")
+		stack_trace("Skillchip doesn't have a holding brain.")
 		return "Skillchip cannot detect viable brain."
 
+	if(!brain_owner)
+		brain_owner = holding_brain.owner
 	// Also should not happen. We're somehow deactivating skillchips in a bodyless brain.
-	if(QDELETED(holding_brain.owner))
+	if(QDELETED(brain_owner))
```

Comment:
``try_deactivate_skillchip`` is called from a bunch of other places and you're only changing one - you should probably do first

```js
if (isnull(brain_owner))
	brain_owner = holding_brain.owner
```

Reply:
It's already being done a few lines above btw lol